python/lsst/obs/pfs/constructFiberFlatTask.py

```diff
@@ -41,6 +42,9 @@ class ConstructFiberFlatConfig(CalibConfig):
         default="DARKTIME",
         doc="Header keyword for time since last CCD wipe, or None",
         optional=True)
+    display = Field(dtype=bool,
+        default=True,
+        doc="Display images?")
```

Comment:
Generally, you don't want operational parameters in the `Config`, because they can change without actually changing the processing.  One option is to move this into the `ArgumentParser`, but that also requires changing the `TaskRunner`.  I suggest you use the 'debug' mechanism, as demonstrated [here](https://github.com/lsst/meas_algorithms/blob/master/python/lsst/meas/algorithms/pcaPsfDeterminer.py#L193).  Then you can put in your code:

    import lsstDebug
    if lsstDebug.Info(__name__).display:
        import lsst.afw.display.ds9 as ds9  # This will change later to be backend-agnostic
        ds9.mtv(...)

To activate it, put in a file called `debug.py` somewhere on your `PYTHONPATH`:

    import lsstDebug
    print "Importing debug settings..."
    def DebugInfo(name):
        di = lsstDebug.getInfo(name)  # N.b. lsstDebug.Info(name) would call us recursively
        if name in (
            "lsst.obs.pfs.constructFiberFlatTask"
            ):
            di.display = True
        return di

    lsstDebug.Info = DebugInfo